src/getHandlers.js

```diff
@@ -0,0 +1,42 @@
+import lifecycle from './lifecycle'
+
+function format (string) {
+  /*eslint-disable no-unused-vars*/
+  const [ _, action, state = 'done' ] = `${ string }`.match(/(\w*)\_\d+[\_\w]*/) || []
+  /*eslint-enable no-unused-vars*/
```

Comment:
oh, nice. Didn't realize you could do that inline. Sad that we can't make a global exception for this pattern 😞